question/bank/viewcreator/templates/modifier_display.mustache

```diff
@@ -0,0 +1,33 @@
+{{!
+    This file is part of Moodle - http://moodle.org/
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template qbank_viewcreator/modifier_display
+    Example context (json):
+    {
+        "displaydata": [
+          {
+                "modifier": "Marc Ghaly",
+                "date": "2 June 2021, 5:32 PM",
+          }
+        ]
+    }
+}}
+
+<span>
```

Comment:
I'd add a class to spam tag in order to add style for future theme developers.

Reply:
`class="modifierclass"` added